bindings/nodejs/examples/how_tos/native_tokens/create.ts

```diff
@@ -0,0 +1,85 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { CreateNativeTokenParams, utf8ToHex } from '@iota/sdk';
+
+import { getUnlockedWallet } from '../../wallet/common';
+
+// The circulating supply of the native token. `100` hex encoded
+const CIRCULATING_SUPPLY = '0x64';
+// The maximum supply of the native token. `100` hex encoded
+const MAXIMUM_SUPPLY = '0x64';
+
+// In this example we will create a native token.
+//
+// Make sure that `example.stronghold` and `example.walletdb` already exist by
+// running the `how_tos/accounts-and-addresses/create-wallet` example!
+//
+// Rename `.env.example` to `.env` first, then run
+// yarn run-example ./how_tos/native_tokens/create.ts
+async function run() {
+    try {
+        // Create the wallet
+        const wallet = await getUnlockedWallet();
+
+        // Get the account we generated with `01-create-wallet`
+        const account = await wallet.getAccount('Alice');
+
+        const balance = await account.sync();
+
+        // We can first check if we already have an alias in our account, because an alias can have many foundry outputs and therefore we can reuse an existing one
+        if (balance.aliases.length > 0) {
```

Comment:
only need to make if length == 0 right?

Reply:
hmm, yeah, you're right ... seems wrong ... though ... I didn't write this code :thinking: Should I fix it?

Reply:
yes please 🙏

Reply:
changed